test/__snapshots__/build-media-variants.jest.js.snap

```diff
@@ -9,50 +9,13 @@ exports[`buildMediaVariants defaults 1`] = `
 .col-mm {
   display: block;
   max-width: 100%;
-  flex-basis: 0;
```

Comment:
We should talk about this change. This changes the behaviour of `.col` but excluding it means we can omit `flex:none` that were previously on `col--n` selectors